app/javascript/components/ShowBookImage.jsx

```diff
@@ -28,11 +29,11 @@ export default function ShowBookImage(props) {
     ) {
       return (
         <Box
-          bg={bookBgColor}
           paddingLeft="s"
           padding="s"
           width="189px"
           height="189px"
+          color={bookTitleColor}
```

Comment:
Can you incude the `color` attribute with the same set of styles that set the background color for the `<a>` element?